run.sh

```diff
@@ -0,0 +1,8 @@
+git checkout wait-button
```

Comment:
can you add a brief doc about what this script is for

Reply:
Was only for debugging, removed.